configs/KONEXH743/config.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        KONEXH743
+#define MANUFACTURER_ID   KONX
+
+#define USE_GYRO
+#define USE_GYRO_CLKIN
+#define USE_ACC
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+
+#define USE_BARO
+#define USE_BARO_BMP388
+
+#define LED0_PIN PC13
+#define LED1_PIN PC10
+
+#define LED_STRIP_PIN PC9
+  
+#define BEEPER_PIN PE10
+#define BEEPER_INVERTED
+
+#define UART3_TX_PIN PD8
+#define UART3_RX_PIN PD9
+
+#define UART4_TX_PIN PA0
+#define UART4_RX_PIN PA1
+
+#define UART5_RX_PIN PD2
+
+#define UART6_TX_PIN PC6
+#define UART6_RX_PIN PC7
+
+#define UART7_RX_PIN PA8
+
+#define UART8_TX_PIN PE1
+#define UART8_RX_PIN PE0
+
+#define I2C2_SCL_PIN PB10
+#define I2C2_SDA_PIN PB11
+
+#define SPI1_SCK_PIN PA5
+#define SPI1_SDI_PIN PA6
+#define SPI1_SDO_PIN PA7
+#define GYRO_1_CS_PIN PB1
+#define GYRO_1_EXTI_PIN PB0
+
+#define SPI2_SCK_PIN PB13
+#define SPI2_SDI_PIN PB14
+#define SPI2_SDO_PIN PB15
+#define GYRO_2_CS_PIN PB12
+#define GYRO_2_EXTI_PIN PE13
+
+#define SPI4_SCK_PIN PE2
+#define SPI4_SDI_PIN PE5
+#define SPI4_SDO_PIN PE6
+#define FLASH_CS_PIN PE4
+
+#define ADC_VBAT_PIN PC0
+#define ADC_CURR_PIN PC3
+
+#define MOTOR1_PIN PB9
+#define MOTOR2_PIN PB8
+#define MOTOR3_PIN PB7
+#define MOTOR4_PIN PB6
+#define MOTOR5_PIN PB4
+#define MOTOR6_PIN PB5
+#define MOTOR7_PIN PB3
+#define MOTOR8_PIN PA15
+
+#define VTX_ENABLE_PIN                PE3
+#define PINIO1_PIN                    VTX_ENABLE_PIN
+#define PINIO1_BOX 40
```

Comment:
Probably want to enable as default
```suggestion
#define PINIO1_BOX 40
#define PINIO1_CONFIG 129
```